src/mixins/date-mixin.ts

```diff
@@ -46,12 +46,15 @@ export default class DateMixin extends Mixins(CommonMixin) {
   createUtcDate (year: number, month: number, day: number, hours: number = 0, minutes: number = 0): Date {
     // use date from server to create a new date in Pacific timezone
     // (this sets the correct tz offset in the new date)
-    const date = new Date(this.getCurrentJsDate.toLocaleString('en-US', { timeZone: 'America/Vancouver' }))
```

Comment:
Since `getCurrentDate` is no longer needed, could you please delete line 15 and 2?